src/data/information-technology-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const informationTechnologyQuiz = [
+  {
+    Question: "What it's the name of the modern computers architecture?",
+    Answer: "Von Neumann Architecture",
+    Distractor1: "Enigma-Turing Architecture",
+    Distractor2: "Micro-ATX Architecture",
+    Distractor3: "CPU-GPU Architecture",
+    Explanation: "John Von Neumann is the creator of the modern computer architecture",
+    Link:"https://www.bbc.co.uk/bitesize/guides/zhppfcw/revision/3"
```

Comment:
Please only use freeCodeCamp resources or wikipedia links.

```suggestion
    Link:"https://en.wikipedia.org/wiki/Von_Neumann_architecture"
```